java/src/Ice/src/main/java/com/zeroc/Ice/UserException.java

```diff
@@ -6,12 +6,6 @@
 
 /** Base class for exceptions defined in Slice. */
 public abstract class UserException extends java.lang.Exception implements Cloneable {
-  public UserException() {}
```

Comment:
Can probably remove Cloneable.

Reply:
Yes, I plan to in a separate PR.

Cloneable also needs to be removed from the local side as-well.